CHANGELOG.md

```diff
@@ -26,6 +26,7 @@
 * [#5357](https://github.com/bbatsov/rubocop/issues/5357): Fix `Lint/InterpolationCheck` false positives on escaped interpolations. ([@pocke][])
 * [#5409](https://github.com/bbatsov/rubocop/issues/5409): Fix multiline indent for `Style/SymbolArray` and `Style/WordArray` autocorrect. ([@flyerhzm][])
 * [#5393](https://github.com/bbatsov/rubocop/issues/5393): Fix `Rails/Delegate`'s false positive with a method call with arguments. ([@pocke][])
+* Fix exception when executing from a different drive on Windows. ([@orgads][])
```

Comment:
This is causing your CI failures. You'll have to add your github handle at the [bottom of the file](https://github.com/bbatsov/rubocop/blob/master/CONTRIBUTING.md#changelog-entry-format) as well.